packages/@ember/-internals/extension-support/lib/data_adapter.js

```diff
@@ -53,18 +53,18 @@ export default EmberObject.extend({
   init() {
     this._super(...arguments);
     this.releaseMethods = emberA();
-  },
 
-  /**
-    The container-debug-adapter which is used
-    to list all models.
+    /**
+      The container-debug-adapter which is used
+      to list all models.
 
-    @property containerDebugAdapter
-    @default undefined
-    @since 1.5.0
-    @public
-  **/
-  containerDebugAdapter: undefined,
+      @property containerDebugAdapter
+      @default undefined
+      @since 1.5.0
+      @public
+    **/
+    this.containerDebugAdapter = getOwner(this).lookup('container-debug-adapter:main');
```

Comment:
Should we make htis lazy (e.g. with a getter or `computed`)?

Reply:
That seems like a possible route.  Went the value route to match the injection as closely as possible.  Happy to go either route